workflows/run_dev_se.nf

```diff
@@ -51,7 +37,7 @@ workflow RUN_DEV_SE {
     // Publish results
     params_str = JsonOutput.prettyPrint(JsonOutput.toJson(params))
     params_ch = Channel.of(params_str).collectFile(name: "run-params.json")
-    time_ch = Channel.of(start_time_str + "\n").collectFile(name: "time.txt")
+    time_ch = Channel.of(params.start_time_str + "\n").collectFile(name: "time.txt")
```

Comment:
How is this getting into params? Seems naively like it would be simpler to just emit it from LOAD_SAMPLESHEET.

Reply:
@simonleandergrimm what Will is referring to here is this line "params.start_time_str". Previously this was in the main workflow like this:
`start_time = new Date()
    start_time_str = start_time.format("YYYY-MM-dd HH:mm:ss z (Z)")`
. Either move this back to the main workflow (in which case you can use `start_time_str`, or keep it in LOAD_SAMPLESHEET (and make sure to emit it).